### DISCUSSION TITLE
Please if possible add regexp2 in httpx. [FTL] Invalid value for match regex option: error parsing regexp: invalid or unsupported Perl syntax: `(?!`

### DISCUSSION THREAD

**Discussion by foorw1nner** [projectdiscovery/httpx#1499] [Q&A]
This error appears every time I try to use a negative lookahead in a regex match, example: (?!apple)
Solutions or alternatives?

Project Discovery, It was identified that the httpx does not support the use of regexp2 library in matching and extracting regex functionalities, -mr and -er. I ask if it is possible to implement regexp2 in httpx for such functions.

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
